2018/python/test_file_generator.py

```diff
@@ -23,6 +23,14 @@ def __init__(self, ts_schema, lar_schema, crosswalk_data):
 		#Loads the filepath configuration. 
 		with open('configurations/test_filepaths.yaml') as f:
 			filepaths = yaml.safe_load(f)
+
+		#Loads geographic configuration file. 
+		with open('configurations/geographic_data.yaml') as f:
+			self.geographic = yaml.safe_load(f)
+
+		#Loading in state codes. 
+		self.state_codes = self.geographic['state_codes']
```

Comment:
If geographic is a class property, there is no need to load state codes as a separate class variable.

Reply:
Sure - this code was written to shorten syntax, but I can refactor so that state codes are not loaded as a separate class variable, if that works best.

Reply:
Actually, we can remove this line of code. We would only need state abbreviations here.